engines/io_uring.c

```diff
@@ -23,6 +23,10 @@
 #include "../lib/types.h"
 #include "../os/linux/io_uring.h"
 
+#ifndef FIO_RANDSEED
```

Comment:
Both here and in libaio - and we have an existing one as well. You should add an extra element to td->rand_seeds[] instead, and have it be initialized in td_fill_rand_seeds_internal() like all the other ones. Then you also get the correct behavior in terms of 'rand_repeat', right now your sequence is always 100% repeatable which is probably not idea. It also means that you can remove some of the init/boiler plate code from libaio and io_uring and keep them in the spot where they belong, outside of the engine themselves.